data/darktableconfig.xml.in

```diff
@@ -1230,13 +1230,6 @@
     <shortdescription>do high quality processing for slideshow</shortdescription>
     <longdescription>same option as for export, but applies to slideshow.</longdescription>
   </dtconfig>
-  <dtconfig prefs="core" section="quality">
```

Comment:
This should be <dtconfig> and we need to keep this entry. This will be an hidden pref.